target/linux/lantiq/files-4.14/arch/mips/boot/dts/FRITZ7362SL.dts

```diff
@@ -0,0 +1,285 @@
+/dts-v1/;
```

Comment:
@danijelt  Please can you consider `SPDX-License-Identifier: GPL-2.0-or-later OR MIT` as in [DTS Checklist](https://openwrt.org/submitting-patches#dts_checklist) ?
> Having MIT as alternative is sometimes preferred by upstream maintainers
and allows sharing that simple code with other projects. We don't really
want multiple DTS versions for the same device.

Reply:
Done.